code/style.css

```diff
@@ -1,4 +1,285 @@
+        justify-content: space-between
+    }
+    /* I am so trying to figuere out why the logo dosen't go the way I need it to so I took it away*/
+    .nav1 {
+        order: 1;
+    }
+    .nav2 {
+        order: 2;
+    }
+    .nav3 {
+        order: 4;
+    }
+    .nav4 {
+        order: 5;
+    }
+    header {
+        justify-content: space-between;
+    }
+}
+
+
+/* =========== HOVER AND STUFF =========== */
+
+a {
+    color: #000000;
+    text-decoration: none;
+    transition: 0.15s;
+    box-shadow: 0px 4px 0px #1732cc;
+}
+
+a:hover {
+    color: #f3b94e;
+    box-shadow: 0px 8px 0px #f19443;
+}
+
+h1,
+h2 {
+    font-family: 'Playfair Display', serif;
+    font-weight: 800;
+    margin: 0;
+}
+
+h1 {
+    font-size: 42px;
+    line-height: 56px;
+    margin-bottom: 16px;
+}
+
+h2 {
+    font-size: 24px;
+    line-height: 32px;
+    margin-top: 8px;
+    /* padding-bottom: 16px; */
+}
+
+.redlink {
+    font-family: 'PT Sans', sans-serif;
+    ;
+    font-weight: 800;
+    color: #000;
+    text-decoration: none;
+}
+
+p {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 30px;
+    color: rgba(0, 0, 0, 0.87);
+    margin: 0;
+    margin-bottom: 16px;
+}
+
+.credits {
+    font-size: 14px;
+    color: rgba(0, 0, 0, 0.20);
+    font-family: 'PT Sans', sans-serif;
+    font-weight: 400;
+}
+
+
+/* ======CONTENT =======*/
+
+.grid-container {
+    display: grid;
+    grid-gap: 48px 20px;
+    margin-bottom: 120px;
+}
+
+
+/* smartphone */
+
+@media only screen and (max-width: 767px) {
+    .grid-container {
+        grid-template-columns: repeat(1, 1fr);
+    }
+    h1 {
+        font-size: 24px;
+        line-height: 36px;
+        margin-bottom: 16px;
+    }
+}
+
+
+/* tablet */
+
+@media only screen and (min-width: 768px) {
+    .grid-container {
+        grid-template-columns: repeat(2, 1fr);
+    }
+}
+
+
+/* desktop */
+
+@media only screen and (min-width: 992px) {
+    .grid-container {
+        grid-template-columns: repeat(4, 1fr);
+    }
+}
+
+.pagecontainer {
+    max-width: 1200px;
+    margin: auto;
+}
+
+.card {
+    padding: 0 0 40px 0;
+}
+
+.grid-container .card {
+    display: flex;
+    flex-direction: column;
+    padding: 0 0 24px 0;
+}
+
+.topstory {
+    display: flex;
+    margin-bottom: 64px;
+    box-shadow: 0px 4px 0px #000000;
+}
+
+main img {
+    object-fit: cover;
+    box-sizing: border-box;
+}
+
+.topstory img {
+    width: 50%;
+    margin-right: 24px;
+}
+
+.topstory--rightcol {
+    width: 50%;
+}
+
+.topstory--linkbox {
+    display: flex;
+    flex-direction: row-reverse;
+}
+
+[class^='card small'] img {
+    width: 100%;
+    height: 160px;
+}
+
+footer {
+    display: flex;
+    justify-content: center;
+    background-color: rgb(241, 220, 224);
+}
+
+
+/* I really don' know how everything fits togheter */
+
+
+/* tablet and smartphone */
+
+@media only screen and (max-width: 992px) {
+    .topstory--rightcol p {
+        height: 0px;
+    }
```

Comment:
Not sure what you wanted to achieve here. This causes your content to overlap (you are not allocating any space for it, but it's still there).  Did you want to hide it? In that case, go with:
`display: none;`